spec/ruby/core/kernel/require_relative_spec.rb

```diff
@@ -40,6 +40,18 @@
     ScratchPad.recorded.should == [:loaded]
   end
 
+  it "loads a path relative to a file base name as specified in an #instance_eval" do
+    Dir.chdir File.dirname(File.join(__dir__, @path)) do
```

Comment:
Can you use `Dir.chdir @abs_dir` here, since it's already computed above and easier to read?

Reply:
Ok.